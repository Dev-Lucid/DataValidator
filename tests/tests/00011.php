<?php
$field = dvr::field('field1')->rule('length','fail_message',4,6);

$out  = '';
list($result,$errors) = $field->test(array('field1'=>'abc'));
$out .= ($result)?'PASSES':'FAIL';
$out .= "\n";
list($result,$errors) = $field->test(array('field1'=>'abcd'));
$out .= ($result)?'PASSES':'FAIL';
$out .= "\n";
list($result,$errors) = $field->test(array('field1'=>'abcde'));
$out .= ($result)?'PASSES':'FAIL';
$out .= "\n";
list($result,$errors) = $field->test(array('field1'=>'abcdef'));
$out .= ($result)?'PASSES':'FAIL';
$out .= "\n";
list($result,$errors) = $field->test(array('field1'=>'abcdefg'));
$out .= ($result)?'PASSES':'FAIL';
$out .= "\n";
list($result,$errors) = $field->test(array('field1'=>'abcdefgh'));
$out .= ($result)?'PASSES':'FAIL';
$out .= "\n";
list($result,$errors) = $field->test(array('field1'=>'abcdefghi'));
$out .= ($result)?'PASSES':'FAIL';
$out .= "\n";
list($result,$errors) = $field->test(array('field1'=>'abcdefghij'));
$out .= ($result)?'PASSES':'FAIL';

file_put_contents($output_path,$out);
?>