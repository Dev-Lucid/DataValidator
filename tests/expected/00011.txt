FAIL
PASSES
PASSES
PASSES
FAIL
FAIL
FAIL
FAIL